index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nature UI</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel = "icon" href = "./gardenin-icon.png" type = "image/x-icon"> 
+    <link href="styles.css" type="text/css" rel="stylesheet"/>
+	<style>
+		body{
```

Comment:
Here, instead of giving styles on the page itself, you can give class name to `body` tag and add those properties in that class.